Move PSU on before setting heating pwm (PID autotune)

GMagician opened the pull request:
This should definitively fix #19056

GMagician committed:
Move PSU on before setting heating pwm

GMagician committed:
PIO-dev unsuitable for CI

GMagician commented:
@thisiskeithb it seems to work

thisiskeithb commented:
> @thisiskeithb it seems to work

It wasn’t confirmed to work by the person that reported it when I added the label. Since it’s been tested and works now, I’ve removed it.

GMagician commented:
> It wasn’t confirmed to work by the person that reported it when I added the label. Since it’s been tested and works now, I’ve removed it.

That's ok, I knew that, I just reported his feedback

thinkyhead committed:
Update test-builds.yml
